File: src/app/templates/java/src/feature-test/java/steps/DummyStepDef.java
package steps;

import io.cucumber.java.PendingException;
import io.cucumber.java.en.Then;
import io.cucumber.java.en.When;

public class DummyStepDef {

    @When("I create {n} useless feature file(s?)")
    public void when_i_create_N_useless_feature_files(Integer n){
        throw new PendingException();
    }

    @Then("I will delete the file")
    public void then_i_will_delete_the_file(){
        throw new PendingException();
    }

    @Then("I waste time")
    public void then_i_waste_time(){
        throw new PendingException();
    }

}

File: src/app/templates/kotlin/base-kotlin-master/src/feature-test/kotlin/steps/DummyStepDef.kt
package steps

import io.cucumber.java.PendingException
import io.cucumber.java.en.Given
import io.cucumber.java.en.Then
import io.cucumber.java.en.When
import java.time.LocalDate

class DummyStepDef {

    @Given("I have {n} useless feature file(s?)")
    fun `I have N useless feature file`(n: Int) {
        throw PendingException()
    }

    @When("I create {n} additional useless feature file(s?)")
    fun `When I create N useless feature file`(n: Int) {
        throw PendingException()
    }

    @Then("I waste time")
    fun `Then I waste time`() {
        throw PendingException()
    }

}


